src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -1012,6 +1012,22 @@ public static void onEmptyLeftClick(EntityPlayer player, ItemStack stack)
         MinecraftForge.EVENT_BUS.post(new PlayerInteractEvent.LeftClickEmpty(player, stack));
     }
 
+    public static net.minecraftforge.event.entity.player.CriticalHitEvent getCriticalHit(EntityPlayer player, Entity target, boolean vanillaCritical, float vanillaDefault)
```

Comment:
That doesn't really make sense, you have a hard-coded `1.5` in the only call to this function. It's not getting that value from anywhere inside vanilla code.